admin-ui/.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
should we mark `.husky` under [.gitignore](https://github.com/GluuFederation/flex/blob/main/admin-ui/.gitignore)?

I think we should not check-in this folder. It can be created using `npm run prepare`

Reply:
Yes, but the pre-commit file is manually created, it's because the repo is sub folder (admin-ui is subfolder) of git root folder @duttarnab 

So we only ignore `.husky/_` folder, wdyt?

Reply:
can we automatically generated  pre-commit file.

![image](https://user-images.githubusercontent.com/32794267/172540309-0dd7712a-a375-4b36-add5-4308e4622b09.png)

Reply:
> Yes, but the pre-commit file is manually created, it's because the repo is sub folder (admin-ui is subfolder) of git root folder @duttarnab
> 
> So we only ignore `.husky/_` folder, wdyt?

@harryandriyan you can still introduce .gitignore in subfolders

Reply:
yes, I mean the `pre-commit` file is manually created, it's because `admin-ui` is in the sub folder of git, it's needed to make husky work, I will find way to automatically handle that @duttarnab